### DISCUSSION TITLE
Vercel serverless functions crashing with 500: INTERNAL_SERVER_ERROR

### DISCUSSION THREAD

**Discussion by DavidKvas** [vercel/vercel#9241] [Help] [answered]
Hi, 

Recently, our vercel serverless functions stopped working with the following 500: INTERNAL_SERVER_ERROR.

If I deploy an old version of our project (which works fine on the prod), it doesn't work either. So it's clear something has changed on the backend.

Unfortunately, there is no error in the vercel web console. 

Vercel function error: https://share.cleanshot.com/S0WSkP27

Test URL: https://staging.curium.app/api/v1/public/api-test

Any help would be much appreciated.

Thanks,

David

**Comment by DavidKvas** [comment]
In the end, I could get some logs from the vercel web console.

I added a package.json file to the API folder to make it work. However, I can't import modules from outside the API folder (which was working fine before).

![CleanShot 2023-01-19 at 10 53 19](https://user-images.githubusercontent.com/808750/213411598-3d2d3873-7287-4f5c-a0a2-3a17119bef58.jpg)

**Comment by odeds-confetti** [comment, marked as answer]
maybe this the answer: https://github.com/orgs/vercel/discussions/1226

I has the same issue added VERCEL_CLI_VERSION=vercel@28.10.2 to env vars fixed it

**Reply by DavidKvas** [reply]
Hey @odeds-confetti,

Yep, that fixed the issue. Thanks so much for pointing me in the right direction.
